src/platform/K32W/Logging.cpp

```diff
@@ -13,6 +13,9 @@
 #define EOL_CHARS "\r\n" /* End of Line Characters */
 #define EOL_CHARS_LEN 2  /* Length of EOL */
 
+#define TIMESTAMP_MAX_LEN_BYTES 10
+#define CATEGORY_MAX_LEN_BYTES 1
```

Comment:
Please document where these magic numbers are coming from.

Reply:
also please use C++ constants `static constexpr` rather than defines.

Reply:
Documented the origin of the numbers and also used "static constexpr". Opened https://github.com/project-chip/connectedhomeip/pull/8451